src/core/auth/authproxy/auth.go

```diff
@@ -140,6 +158,9 @@ func (a *Auth) tokenReview(sessionID string) (*k8s_api_v1beta1.TokenReview, erro
 
 // OnBoardUser delegates to dao pkg to insert/update data in DB.
 func (a *Auth) OnBoardUser(u *models.User) error {
+	if !a.CaseSensitive {
+		u.Username = strings.ToLower(u.Username)
+	}
```

Comment:
It is better to extract a function so that the a.CaseSesnsitive appear everywhere.

Reply:
If we do that we still need to call this function in different funcs in `core/auth/authproxy/auth.go` , so I don't think it we need to do it as this piece of code is quite short.

Reply:
fixed